### PR TITLE
silent install of Firefox 41

### PR DESCRIPTION
**Pull request by jalewis**

### CODE REVIEW

**Comment by jbremer** [comment]
Nice!

File: firefox_41.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#silent install of Firefox 41
+#https://download-installer.cdn.mozilla.net/pub/firefox/releases/41.0.2/win32/en-US/Firefox%20Setup%2041.0.2.exe
+#file was renamed Firefox_Setup_41.0.2.exe to avoid issues with spaces.
+
+rom vmcloak.abstract import Dependency
```
Comment (booboule) [inline firefox_41.py]: I guess there's a small typo. Missing 'f'. :)

**Comment by booboule** [comment]
By the way, the file is probably located in the wrong directory : the git root directory instead of vmcloak/dependencies/ dir.

**Comment by jbremer** [comment]
Oh, thanks for letting me know! I hadn't tested it myself yet.

**Comment by jbremer** [comment]
Hopefully all is resolved with the following commit https://github.com/jbremer/vmcloak/commit/dd7fc10fd7c1b9592f98abda910ddf9c982a42a5.

**Pull request merged**